docs/source/stochastic_training/index.rst

```diff
@@ -7,6 +7,37 @@ scalability. It is built on top of DGL and PyTorch.
 This tutorial introduces how to enable stochastic training of GNNs with
 GraphBolt.
 
+Overview
+^^^^^^^
+
+.. image:: ../../_static/graphbolt_overview.png
+  :width: 700
+  :alt: Graphbolt Overview
+
+GraphBolt seamlessly integrates with the PyTorch `datapiple <https://pytorch.org/data/beta/torchdata.datapipes.iter.html>`_, where each stage's output is just the input of its successor, streamlining data loading and preprocessing for GNN training, validation and testing.
+By default, GraphBolt provides a collection of built-in datasets and exceptionally efficient implementations of datapipes for common scenarios, which can be summarized as follows:
+
+1. **Mini-Batch Sampler:** Randomly selects a subset (nodes, edges, graphs) from the entire training set to calculate embeddings.
+
+2. **Negative Sampler:** In link prediction tasks, samples non-existing edges as negative examples to train the GNN in distinguishing between positive and negative edges.
+
+3. **Subgraph Sampler:** Utilizes the output from mini-batch sampling to generate neighbor nodes up to a specified depth or distance.
```

Comment:
Does it have to be neighbor nodes? What about induced subgraph? Can you rephrase it and focus on just about *generating subgraph*.